src/netex_france/offer.rs

```diff
@@ -119,6 +122,46 @@ impl<'a> OfferExporter<'a> {
             .attr("version", "any")
             .append(Self::generate_distance())
             .append(Self::generate_route_ref(&journey_pattern.route_id))
+            .append(points_in_sequence)
+            .build()
+    }
+
+    fn export_stop_points_in_journey_pattern(
+        &self,
+        journey_pattern_idx: Idx<JourneyPattern>,
+    ) -> Vec<Element> {
+        let vehicle_journey = &self.model.vehicle_journeys[journey_pattern_idx];
+        vehicle_journey
+            .stop_times
+            .iter()
+            .map(|stop_time| {
+                self.export_stop_point_in_journey_pattern(&vehicle_journey.id, stop_time)
+            })
+            .collect()
+    }
+
+    fn export_stop_point_in_journey_pattern(
+        &self,
+        vehicle_journey_id: &'a str,
+        stop_time: &'a StopTime,
+    ) -> Element {
+        Element::builder(ObjectType::StopPointInJourneyPattern.to_string())
+            .attr(
+                "id",
+                Self::generate_stop_sequence_id(
+                    &vehicle_journey_id,
+                    stop_time.sequence,
+                    ObjectType::ScheduledStopPoint,
```

Comment:
```suggestion
                    ObjectType::StopPointInJourneyPattern,
```

Reply:
Done and I updated the fixtures with it.